app/code/Magento/AsynchronousOperations/Model/BulkNotificationManagement.php

```diff
@@ -119,6 +123,7 @@ public function getAcknowledgedBulksByUser($userId)
                 'main_table.uuid = acknowledged_bulk.bulk_uuid',
                 []
             )->addFieldToFilter('user_id', $userId)
+            ->addFieldToFilter('user_type', UserContextInterface::USER_TYPE_ADMIN)
```

Comment:
Here I have some doubts.
My goal initially was to allow to Admin users see Bulk Action of everybody if he is a Master user or he has an access to particular usertypes actions.
This change is restrict user to admin type only.
Also function name getAcknowledgedBulksByUser is not pointing that its restricted only to admins (I understand that code used only in admin now, but if follow service decomposition, this should not be restricted on code leverl)